TodoListService/Models/ClaimInfo.cs

```diff
@@ -7,8 +7,6 @@ namespace TodoListService.Models
 {
     public class ClaimInfo
     {
-        public ClaimInfo() { }
```

Comment:
Is it common not to have an empty constructor? is this one available implicetely? Why remove this one?